wagtail/contrib/search_promotions/templates/wagtailsearchpromotions/list.html

```diff
@@ -18,7 +18,9 @@ <h2><a href="{% url 'wagtailsearchpromotions:edit' query.id %}" title="{% trans
                 </td>
                 <td>
                     {% for searchpick in query.editors_picks.all %}
-                        <a href="{% url 'wagtailadmin_pages:edit' searchpick.page.id %}" class="nolink">{{ searchpick.page.title }}</a>{% if not forloop.last %}, {% endif %}
+                        {% if searchpick.page %}
+                            <a href="{% url 'wagtailadmin_pages:edit' searchpick.page.id %}" class="nolink">{{ searchpick.page.title }}</a>
+                        {% endif %} {% if not forloop.last %}, {% endif %}
```

Comment:
where do external pages get shown?

Reply:
In this implementation they're not differentiated from normal promoted search results. So, they're just shown at the top of the page with the normal promoted search results. They're really no different from the normal promoted search results other than the fact that they go to an external page. Certainly we could discuss changing this.

Reply:
I see - thanks for explaining!

Reply:
@bbusenius I think you're describing how they get shown on the frontend, while @tomdyson is asking about the admin listing page. It looks like the listing here is missing something to output `{{ searchpick.external_page_title }}` when `searchpick.page` is blank - otherwise the entry gets skipped over entirely (but the trailing comma still gets output).

Reply:
Thank you @gasman! Yes, I realized my response was off during my commute last night. On the front end people can implement it however they'd like. In the Wagatail Bakery, I just showed it as part of the normal search results but none of that is included as part of this pull request of course.